dev/integration_tests/abstract_method_smoke_test/android/app/src/main/kotlin/com/example/abstract_method_smoke_test/MainActivity.kt

```diff
@@ -41,7 +44,7 @@ class MainActivity : FlutterActivity() {
 
         // Triggers the Android keyboard, which causes the resize of the Flutter view.
         // We need to wait for the app to complete.
-        MethodChannel(flutterView, "com.example.abstract_method_smoke_test")
+        MethodChannel(flutterEngine.getDartExecutor(), "com.example.abstract_method_smoke_test")
```

Comment:
There has been an API change to `flutterEngine.getDartExecutor().getBinaryMessenger()`. Should we make that change now? Or should we come back later and update?

Reply:
I'd prefer to check this in and then change it again.

Reply:
will follow up with that while the device lab tests are running.